tests/conftest.py

```diff
@@ -206,6 +207,10 @@ def rand_one_dut_hostname(request):
         dut_hostnames = random.sample(dut_hostnames, 1)
     return dut_hostnames[0]
 
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
This might be more appropriate for a dual ToR specific conftest file

Reply:
Good suggestion! I will update. Thanks!